source/administration/config-settings.rst

```diff
@@ -727,6 +727,35 @@ Test Connection
 
 Ensures that the user can access the server and that the settings are valid.
 
+Enable Document Search by Content
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+File search is available now in Mattermost Cloud, and in Mattermost Self-Managed from v5.35, with mobile support coming soon.
+
+**True**: When **true**, supported document types are searchable by their content. Install `these dependencies <https://github.com/sajari/docconv#dependencies>`__ to extend content searching support to include DOCX, RTF, and PAGES files. 
+
+**False**: Supported document types aren't searchable by their content. When document content search is disabled, users can search for files by file name only.
+
++---------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"XXX": true`` with options ``true`` and ``false``.                      |
```

Comment:
@jespino - What is the name of this configuration setting in the ``config.json`` file?